aptible/endpoints.go

```diff
@@ -0,0 +1,130 @@
+package aptible
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/reggregory/go-deploy/client/operations"
+	"github.com/reggregory/go-deploy/models"
+)
+
+// CreateEndpoint() creates Vhost API object + provision operation on the app.
+func (c *Client) CreateEndpoint(app_id int64, attrs map[string]interface{}) (*models.InlineResponse2019, error) {
```

Comment:
```suggestion
func (c *Client) CreateEndpoint(app_id int64, attrs map[string]interface{}) (*models.InlineResponse2019, error) {
```
Instead of `map[string]interface{}` can we either split out the individual arguments or make a specific struct for this? That way we get the type safety on the inputs? `interface{}` is generally a code smell that we try to avoid if we can, and the casts in the body below will panic if the inputs aren't correct. See https://play.golang.org/p/_fTCqtIibA- for the example panic.